pkg/handlers/docker/mutation/customimage/inject_worker_test.go

```diff
@@ -1,21 +0,0 @@
-// Copyright 2023 D2iQ, Inc. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package customimage
-
-import (
-	"testing"
-
-	"github.com/d2iq-labs/capi-runtime-extensions/common/pkg/capi/clustertopology/handlers/mutation"
-	"github.com/d2iq-labs/capi-runtime-extensions/pkg/handlers/docker/mutation/customimage/tests"
-)
-
-func TestWorkerGeneratePatches(t *testing.T) {
```

Comment:
how are the patches can be unit tested after this refactor?

Reply:
Only via the meta mutator tests - but keeping the test files in the `tests` package under this handler.